packages/israel/israel.svg

```diff
@@ -0,0 +1,45 @@
+     title="HaẔafon"
```

Comment:
Could you use `name` attribute (instead of `title`) with English location's name (without special letter `Ẕ`, `H̱`...) please?